### DISCUSSION TITLE
Suggestion: show in overview of packages at which volume they are installed

### DISCUSSION THREAD

**Discussion by Sir-Bacon** [007revad/Synology_app_mover#13] [Ideas]
Suggestion: perhaps in the overview as shown in the first screenshot, in the listing of installed packages split between volumes where they are installed.

Background: I recently installed on my DS918+ 2 SSD's next to my original HDD's. Some packages I have installed at the new volume or moved already, but not all. I want to move the remainder as well to see if I can get my HDD's to spin down.
Knowing which package is still to move would be helpful in selecting them.

**Comment by 007revad** [comment]
I'm working a new version of this script that shows which volume each package is installed on, sorted by volume number.

![updated_list](https://github.com/007revad/Synology_app_mover/assets/39733752/34176599-5212-40a8-aa43-241759e7d91c)

**Reply by Sir-Bacon** [reply]
Yep, that looks great! looking forward to the next version.
